paper-api/src/main/java/org/bukkit/inventory/InventoryView.java

```diff
@@ -321,8 +321,8 @@ default net.kyori.adventure.text.Component title() {
     /**
      * Gets the menu type of the inventory view.
      *
-     * @return the menu type of the inventory view
+     * @return the menu type of the inventory view or null if not applicable
```

Comment:
rather than just stating null if not applicable might be smart to list the cases where null is returned here as far as I'm aware this is the case only player and animal menus, but it might be worth poking around a bit more to make sure that I'm 100% right about this.  Might just be nice to have a little paragraph not in the return statement being more involved on why this method might return null